dataAccessObjects.py

```diff
@@ -56,6 +56,15 @@ def checkBluetoothIdUsed(self,bluetoothId):
             #TODO raise exception
             return None
 
+    def getUserNickname(self,userId):
+        sql = 'select user_nickname from user where user_id = {userId}'.format(userId=userId)
+        ret = self.db.query(sql)
+        if len(ret):
+            return ret[0][0]
```

Comment:
@chenyang14 have you ever checked the data type of the query result is str(), because the original SQL result will cast to str() type